pkg/sql/logictest/testdata/logic_test/alter_table_mixed_19.2_20.1

```diff
@@ -171,7 +171,7 @@ CREATE TABLE regression_47110(a TIME)
 statement error type TIMETZ is not supported until version upgrade is finalized
 ALTER TABLE regression_47110 ADD COLUMN b TIMETZ
 
-statement error type TIMETZ is not supported until version upgrade is finalized
+statement error all nodes are not the correct version for alter column type general
```

Comment:
Is this error message correct to be changed

Reply:
I think what happens here is ALTER COLUMN TYPE is disabled for general type conversions including this one until all versions are upgraded - this error appears before we actually use the type TIMETZ so we don't reach that error.

Reply:
I'd just delete this test then.